src/Events/ManuallyIdentified.php

```diff
@@ -4,7 +4,21 @@
 
 use Tenancy\Tenant\Events\Event;
 
-class ManuallyIdentified extends Event
+class ManuallyIdentified
 {
-    
+    /**
+     * @var Tenant
```

Comment:
Tenant|null 
:)

Reply:
I removed that hint; because we don't actually know what will be passed. 
It could be a model that isn't related to T/T at all as well...